finesse/gui/interferometer_monitor.py

```diff
@@ -0,0 +1,204 @@
+"""Panel and widgets related to monitoring the interferometer."""
+from copy import deepcopy
+from typing import Optional
+
+from PySide6.QtCore import Qt
+from PySide6.QtWidgets import QGridLayout, QGroupBox, QLabel, QLineEdit
+
+from .led_icons import LEDIcon
+
+
+def get_vals_from_server() -> dict:
+    """Placeholder function for retrieving interferometer properties.
+
+    Returns:
+        data_table: A dictionary containing the physical properties being monitored
+    """
+    data_table = {
+        "PSF27 Temp": [28.151062, "deg. C"],
+        "Cryo Temp": [0.0, "deg. K"],
+        "Blackbody Hum": [2.463968, "%"],
+        "Source Temp": [70.007156, "deg. C"],
+        "Aux Volt": [6.285875, "V"],
+        "Aux Curr": [0.910230, "A"],
+        "Laser Curr": [0.583892, "A"],
+    }
+    return data_table
```

Comment:
I feel like the data structure isn't quite right here, because you're effectively using keys as values and the lists contain two objects of different types. How about defining a dataclass for this, with `name`, `value` and `unit` fields?

Reply:
Or how about using [`pint`](https://pint.readthedocs.io/en/stable/)? Dealing with this properly is the whole point of the tool.

Reply:
The thing is that we're not actually doing anything with the values other than displaying them, so I'm not sure `pint` would help us. It would be different if we wanted to convert between unit types or do dimensional analysis etc.